.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "nifxml"]
 	path = nifxml
-	url = https://github.com/niftools/nifxml.git
+	url = https://github.com/aperture-it/nifxml.git
```

Comment:
You should have left the .gitmodules changes uncommitted in your working directory, as obviously we're not going to change the url away from niftools.  We'll obviously get the required changes merged to nifxml before merging this.